index.html

```diff
@@ -342,18 +342,24 @@
       var mutationCount = 0;
 
       function translateTextNode(node, info) {
-        if(langify.settings.observeCustomContents === false || !node.textContent || node.textContent.trim().length === 0) {
+        if (langify.settings.observeCustomContents === false || Object.keys(customContents_text).length === 0) {
```

Comment:
This part is where I got the biggest improvement from, specifically:
```javascript
Object.keys(customContents_text).length === 0
```
There was a case where `observeCustomContents` was enabled but the `customContents_text` was empty so it didn't make any sense to execute the code further.

I can see two ways to go with this:
1. Create the `isObjectEmpty` utility function and use this check inside each translation function that accesses `customContents_` before doing any other logic.
2. Assuming all of the `customContents_` objects can be filled only during the script initialisation, check if they have any translations inside and if they don't, just flip the corresponding setting inside `langify.settings.` to `false`.